engine/server/server.go

```diff
@@ -407,13 +407,6 @@ func (s *DaggerServer) VerifyClient(clientID, secretToken string) error {
 	return nil
 }
 
-func (s *DaggerServer) UnregisterClient(clientID string) error {
-	s.clientIDMu.Lock()
-	defer s.clientIDMu.Unlock()
-	delete(s.clientIDToSecretToken, clientID)
-	return nil
-}
-
```

Comment:
:eyes: what's the logic for the removal of this? Doesn't this cause the map to grow forever?

I'm *guessing* this was used when debugging https://github.com/dagger/dagger/issues/7334? Disabling still allows the tests to pass.

Reply:
Oh sorry I should have mentioned this in the commit message. There's context for why this `UnregisterClient` method was added recently here: https://github.com/dagger/dagger/pull/7213/commits/df0753150527377c2fb9f8f0f996e08ce1fd2b61

Basically, going back to random IDs invalidates all of those considerations so we can just remove this once again.